SeaSharpe_CVGS/SeaSharpe_CVGS/Controllers/GameController.cs

```diff
@@ -12,7 +15,15 @@ public class GameController : Controller
     {
         private ApplicationDbContext db = new ApplicationDbContext();
 
+        //Dictionary containing ESRB ratings
+        public static List<string> esrbList = new List<string>
+            {
+                {"EC"},{"E"},{"E10"},{"T"},{"M"},{"AO"}
```

Comment:
You could also make a selectlist like this: 
```csharp
    var selectList = new SelectList((IEnumerable)dict, "Key", "Value", "SelectedKey");
```
inspired by: http://stackoverflow.com/a/21307486

The selected key part is optional but could be used to keep the users search preferences from a previous post.